docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Since <code>data_dir</code> is a <code>Path()</code> instance, you can just do:</p><pre class="ql-syntax" spellcheck="false">files = data_dir.glob('*/*_dem.tif') 
</pre><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>